src/components/button/button-test.stories.tsx

```diff
@@ -203,7 +203,7 @@ export const generateButtonsNoWrapStory = (buttonType: ButtonTypes) => {
   return BUTTON_SIZES.map((size) => {
     return (
       <div key={`${buttonType}-${size}`}>
-        <Box style={{ width: 100 }}>
+        <Box width="100px">
```

Comment:
**question:(non-blocking)** this `100` is definitely translated to a pixel value?